bin/cli.js

```diff
@@ -161,139 +172,147 @@ function invoke(env) {
       `--stub [<relative/path/from/knexfile>|<name>]`,
       'Specify the migration stub to use. If using <name> the file must be located in config.migrations.directory'
     )
-    .action((name) => {
-      const opts = commander.opts();
-      opts.client = opts.client || 'sqlite3'; // We don't really care about client when creating migrations
-      const instance = initKnex(env, opts);
-      const ext = getMigrationExtension(env, opts);
-      const configOverrides = { extension: ext };
-
-      const stub = getStubPath('migrations', env, opts);
-      if (stub) {
-        configOverrides.stub = stub;
+    .action(async (name) => {
+      try {
+        const opts = commander.opts();
+        opts.client = opts.client || 'sqlite3'; // We don't really care about client when creating migrations
+        const knex = await initKnex(env, opts);
+        const ext = getMigrationExtension(env, opts);
+        const configOverrides = { extension: ext };
+
+        const stub = getStubPath('migrations', env, opts);
+        if (stub) {
+          configOverrides.stub = stub;
+        }
+
+        const createdName = await knex.migrate.make(name, configOverrides);
+        success(color.green(`Created Migration: ${createdName}`));
+      } catch (error) {
+        console.log(error);
+        exit();
```

Comment:
This change seems unrelated to native ESM support.  (It's just updating the code to use `async` / `await` syntax, correct?)